docs/csharp/language-reference/operators/lambda-expressions.md

```diff
@@ -44,44 +44,57 @@ When you use method-based syntax to call the <xref:System.Linq.Enumerable.Select
   
 ## Expression lambdas
 
-A lambda expression with an expression on the right side of the `=>` operator is called an *expression lambda*. Expression lambdas are used extensively in the construction of [expression trees](../../programming-guide/concepts/expression-trees/index.md). An expression lambda returns the result of the expression and takes the following basic form:
+A lambda expression with an expression on the right side of the `=>` operator is called an *expression lambda*. An expression lambda returns the result of the expression and takes the following basic form:
```

Comment:
Removed that sentence as it breaks the flow of the paragraph; moreover, that information is mentioned earlier in the article.